File: benchmarks/src/lua/count_set_bits.lua
local n=100000;local total=0;local start=os.clock()
for i=1,n do local x=i
 while x>0 do total=total+(x%2);x=math.floor(x/2) end end
local finish=os.clock();print("Result: "..total);print(string.format("Time: %.0fms",(finish-start)*1000))

File: benchmarks/src/nodejs/count_set_bits.js
const n=100000;let total=0;const start=Date.now();
for(let i=1;i<=n;i++){let x=i;while(x>0){total+=x%2;x=Math.floor(x/2)}}
const end=Date.now();console.log('Result: '+total);console.log('Time: '+Math.floor(end-start)+'ms')